app/uk/gov/hmrc/gform/config/ConfigModule.scala

```diff
@@ -54,6 +55,8 @@ class ConfigModule(playBuiltInsModule: PlayBuiltInsModule) {
     val controllerConfigs: TypeSafeConfig = typesafeConfig.as[TypeSafeConfig]("controllers")
   }
 
+  val getAvailableLanguages: Map[String, Lang] = Map("english" -> Lang("en"), "cymraeg" -> Lang("cy"))
```

Comment:
`getAvailableLanguages` name is really weird. Wouldn't `availableLanguages` be better?